### DISCUSSION TITLE
More reactions

### DISCUSSION THREAD

**Discussion by rach-sc** [community/community#10240] [Pull Requests]
On pull requests you can add 'Reactions' to reviewer comments. These are limited to: 👍 👎 😁 🎉 😕 ❤️ 🚀 👀

Can we expand this to pick from the full set of emojis?

Thanks!

**Comment by AmandaHassoun** [comment]
I don't have a valuable answer I just want to bump this up because I've been waiting for this feature for tooo long!!

**Comment by serious-angel** [comment]
Thank you for raising the issue! I believe the reason is to try to keep it more organized and comprehensive, also to keep it more neat and tidy in general considering that removing those is not possible even if compared with wonderful lovely Discord:

<img width="447" alt="chrome_•_Discord__#off-topic__rSteam_-_Google_Chrome_2023-11-06_13-38-25_Z_RUkBmeJr" src="https://github.com/community/community/assets/25136754/0bbb12b3-9636-48bd-8e94-9255d5d8c3d9">

Yet, I also believe these would be interesting: ✨, ⭐, 🥇, 🌱, 🌙, ☀️, 🪐, ☄️, 🗺️, 🧭, 📡, 🛰️, 🔭, 💫...

**Comment by kj800x** [comment]
Some of the emoji names in the current set are also highly misleading at times. I have seen :confused: be used to express bittersweet emotions and sorrow such as when a large project is finally sunset. I've also seen :laugh: be used to express joy, which is very different from laughter. Having a PR that makes someone happy is very different from someone laughing at my PR. With the recent push for more communication to be done online, it's more important than ever that people don't misunderstand, and emojis are rife for misunderstanding especially when limited so much.

**Comment by anina-hitt** [comment]
how do we get this prioritized

**Comment by Arugin** [comment]
We need more reactions.

**Comment by jorisw** [comment]
I would especially like to see ✅ so that I can express having processed individual comments (on comments)

**Reply by benjub** [reply]
Probably one of the most useful reaction emojis: this would allow to avoid cluttering threads with comments merely saying "Done".

**Comment by benjub** [comment]
Looks related to #69934. I'll repost my comment from there here:

There are merits in having a very limited list of reaction emojis: the more emojis you add to denote subtler and subtler nuances, the less of a universal language it becomes. You have to explain newcomers to a project when to use "thumbs up" and when to use "plus", for instance (to take the example of https://github.com/orgs/community/discussions/69934#discussioncomment-9534999 above), you take the risk of different people using them with different implicit meanings, you trigger endless questions: how to compare a comment having received 3 "thumbs up" and 2 "pluses" with a comment having received 4 "thumbs up" and 1 "plus" (and what if a given person reacted with both emojis...), etc.

On the other hand, I think that a "Done" reaction emoji (probably a checkmark) is really missing: its meaning is clear, and this would avoid cluttering threads with comments merely saying "Done".

**Comment by xiankaing** [comment]
Can we at least have :pray: (prayer hands) https://github.com/orgs/community/discussions/38201 
Expressing thank you is so common

**Comment by mejtxd** [comment]
more reactions would be awsome!

**Comment by pouyarad** [comment]
Would really appreciate being able to react with ➕ emoji!

**Comment by supervoidcoder** [comment]
please add :shipit:  i NEED it as a reaction

**Reply by serious-angel** [reply]
> The GitHub devs use HUBOT which is a script that listens to IRC messages and adds useful (or just humorous) behavior. Part of hubot is `shipit.coffee` which listens for "ship it" (or similar) and adds "motivational" pictures. One of these pictures is a "squirrel mobster", so I'm sure that the GitHub devs are used to seeing him on a daily basis, associated with the phrase "ship it"...
> 
> This was added by Justin Palmer of the GitHub staff, on Apr 18, 2012, in this commit: https://github.com/github/hubot-scripts/commit/247310f83e8f0a33230c4a2ceb5e68ca86006e18
> 
> [Source](<https://www.quora.com/On-GitHub-what-is-the-significance-of-the-Ship-It-squirrel/answer/Buck-Evan-1?comment_id=421348453&comment_type=2>)

\---

> ship it - Display a motivation squirrel
> Source: https://github.com/github/hubot-scripts/commit/91b33a64d2be811edc6430f2377eb42b4e8037c5
